packages/deployment/app/cloudformation.template.yml

```diff
@@ -651,10 +651,17 @@ Resources:
 {{/if}}
       Runtime: {{# if this.runtime}}{{this.runtime}}{{else}}nodejs8.10{{/if}}
       Timeout: {{this.timeout}}
+    {{# if this.deadletterqueue}}
```

Comment:
This seems unmaintainable.  But is this the right way to add a config to the lambda CF template?